AmazonChimeSDK/AmazonChimeSDK/audiovideo/DefaultAudioVideoController.swift

```diff
@@ -18,20 +18,23 @@ import Foundation
     private let clientMetricsCollector: ClientMetricsCollector
     private var videoClientController: VideoClientController
     private let videoTileController: VideoTileController
+    private let appLifecycleObserver: AppLifecycleObserver
     private var primaryMeetingPromotionObserver: PrimaryMeetingPromotionObserver?
 
     public init(audioClientController: AudioClientController,
                 audioClientObserver: AudioClientObserver,
                 clientMetricsCollector: ClientMetricsCollector,
                 videoClientController: VideoClientController,
                 videoTileController: VideoTileController,
+                appLifecycleObserver: AppLifecycleObserver,
```

Comment:
The `DefaultAudioVideoController` doesn't use this for anything right? Can we instead start and stop the lifecycle observer in the `init` and `deinit` functions of the `DefaultMeetingSession` so that we have observability even before the audiovideo.start() is called?

Reply:
Discussed offline. We will keep it as is and add an app state attribute events.